src/formatDate.js

```diff
@@ -9,6 +9,35 @@
  */
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const oldSeparator = fromFormat.pop();
+  const newSeparator = toFormat.pop();
+
+  const spliteOldDate = date.split(oldSeparator);
```

Comment:
There is a typo in the variable name `spliteOldDate`. It should be `splitOldDate` to accurately reflect its purpose.